release.nix

```diff
@@ -59,13 +65,16 @@ let
       PKG_IGNORE_TAG = 1;
       buildPhase = ''
         cp ${./package.json} package.json
+        mkdir -p output/prebuilds
```

Comment:
Does this need `output/prebuilds`? Why not just `./prebuilds`? The `./` is the working directory of the build. As long as this directory doesn't conflict with anything in the `./` which I believe is the produced source files from the upstream derivations.

Remember this is an overridden derivation, the `$src` has already unpacked into the current directory.

Reply:
When I use `./` instead of a temp directory, the result included all of the src files.
I also feel that creating and using a temp directory makes the process more explicit.

Reply:
Is that because of:

```
--public-packages "*" \
```

Is the `*` picking up everything?

Reply:
If you must use a a special directory for this. I'd say use `pkg` as the special directory for `pkg` related building.

Reply:
Ok, I can change it to pkg.
as for `--public-packages "*" \` is just flagging everything is a public package since non public packages are included as bytecode by default.. This breaks with the `--no-bytecode` flag.

Reply:
Using pkg 5.3.1 doesn't fix the issue for leveldown prebuilds, neither does adding them as assets.
this seems to be an issue with pkg itself, I've posted an issue at https://github.com/vercel/pkg/issues/1264.